api/src/main/java/org/openmrs/module/labonfhir/api/event/LabCreationListener.java

```diff
@@ -27,12 +39,17 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
+
 import org.openmrs.module.fhir2.api.search.param.TaskSearchParams;
 
 public abstract class LabCreationListener implements EventListener {
 
 	private static final Logger log = LoggerFactory.getLogger(EncounterCreationListener.class);
 
+	private static final String MFL_LOCATION_IDENTIFIER_URI = "http://moh.bw.org/ext/location/identifier/mfl";
+
+	private static final String MFL_LOCATION_ATTRIBUTE_TYPE_UUID = "8a845a89-6aa5-4111-81d3-0af31c45c002";
```

Comment:
can we meake these configurable via GP?? we would not need to hard code them

Reply:
Sure @mozzy11. I just needed to test this out first.